### PR TITLE
implement `psKill` function

### PR DESCRIPTION
**Pull request by mooyoul**
i just found that exported `psKill` function is actually same as `psLookup` function.
so this pr implements `psKill` function.

btw tests seems marked as todo, is it okay to go?
i will open another pr if you don't have any plan to change specs :)

Thanks for your amazing job. It was really helpful for me (yeah your blog post was also helpful haha)
Cheers 🍻

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -24,7 +24,20 @@ export function psKill (options = { command: '' }) {
       if (error) {
         return reject(error)
       }
-      return resolve(result)
+
+      const pid = result && result.pid
+
+      if (!pid) {
+        return reject(new Error('Could not found target process to kill'))
```
Comment (adieuadieu) [inline src/utils.js]: `Could not found target process to kill` --> `The target process does not appear to be running.`

File: src/utils.js [hunk 2]
```diff
@@ -24,7 +24,20 @@ export function psKill (options = { command: '' }) {
...
+      }
+
+      return ps.kill(pid, (_e) => {
```
Comment (adieuadieu) [inline src/utils.js]: I think this violates at least one ESLint rules: no-underscore-dangle. Could you rename it to `killError` please?

**Comment by adieuadieu** [comment]
Actually.. a question: Is the ability to kill the Headless Chrome process important for you? What's your use case? If you need it, I want to make sure it's available in the upcoming v1.0 release.

**Comment by bluepeter** [comment]
> Actually.. a question: Is the ability to kill the Headless Chrome process important for you?

It may be from my perspective? At least with AWS Lambda, I am constantly getting weird memory management side-effects where, for example, a Lambda process (that invoked a separate child process for PhantomJS) times out... for the rest of the life of that Lambda container, all future invocations timeout. Somehow, the child process refuses to die. Could be that some kill availability w/ headless Chrome might ameliorate this sort of issue from cropping up there too.

**Comment by adieuadieu** [comment]
Thanks for the feedback, @bluepeter!

**Comment by mooyoul** [comment]
Ah, I didn't see develop branch haha Okay, I'll refer that branch for futher contribute ;)

and your question... yeah, I think this is helpful for resource (in Lambda Container) management.
Previously i used phantomjs to crawl bunch of websites on AWS Lambda, but sometimes PhantomJS  was gone. When PhantomJS stuck, Entire lambda code was failed. So i need to force kill of PhantomJS process and restarted PhantomJS process to continue remaining task processing.

Well, Chrome (with Headless mode) seems stable till now, but it would be nice if we can handle chrome process.

**Comment by mooyoul** [comment]
@adieuadieu 
Yeah, I already checked your comments and working on that. 
also, I switched to develop branch) to follow latest development progress.

I'll update further details soon (maybe tomorrow). Thanks! :)

**Comment by adieuadieu** [comment]
Hi @mooyoul Thank you for raising this PR. Given that the _develop_ branch includes support for killing a running Chrome instance and that the _develop_ branch will be merged into _master_, I'm going to close this PR for now. Of course, I welcome any future PRs from you and don't mean to discourage you from making contributions!

**Pull request closed without merging**